tests/ldap_input/assert_show_output.py

```diff
@@ -0,0 +1,20 @@
+"""
+Module holding the correct values for show CLI command outputs for the ldap_test.py
+"""
+
+show_ldap_global="""\
+BIND DN                       BIND PASSWORD      BIND TIMEOUT    VERSION  BASE DN              PORT    TIMEOUT
+----------------------------  ---------------  --------------  ---------  -----------------  ------  ---------
+cn=ldapadm,dc=test1,dc=test2  password                      3          3  dc=test1,dc=test2     389          2
```

Comment:
what are the units for BIND TIMEOUT and LDAP TIMEOUT?

Reply:
seconds

Reply:
Is it possible to mention in the header (secs)?